### ISSUE TITLE
define in .Xresources

### ISSUE THREAD

**Issue by Fig-R** [DaveDavenport/rofi#755]
:arrow_right: First read the [guidelines](https://github.com/DaveDavenport/rofi/blob/next/.github/CONTRIBUTING.md)! :arrow_left:

## Version

`Version: 1.4.2`

## Configuration

[gist](https://gist.github.com/Fig-R/c06f4115def73333d6223ef9b269b4dc)

## Launch Command

```rofi -show drun```

## Steps to reproduce

* use `.Xresources` file in my gist
* `-color-window, -color-normal, -color-active` is broken after session start
* `xrdb .Xresources`
* `-color-window, -color-normal, -color-active` is good

Thanks for your help!

**Comment by DaveDavenport** [comment]
This bug report is very hard to follow? 

Your Xresources file only works after doing xrdb .Xresources?

**Comment by Fig-R** [comment]
Yes, before `xrdb .Xresources`,  rofi is broken ([img](https://imgur.com/a/Lu52U)), `rofi -help` display
```	
-color-active [string]                  Color scheme for active row
        active, background, active, active, background (XResources)
```
and after, he's working
```
-color-window [string]                  Color scheme window
        #1D1D26, #FFCC66, #FFCC66 (XResources)
```

Sorry if it's not a bug or if my bug report is poorly explained.

**Comment by DaveDavenport** [comment]
This is not a rofi bug,  rofi just reads the values as stored in the X-server. (it does not read the .xresources file)

Who loads the values on login? I don't know. but that might not be loading them correctly.

You can query what is in the Xserver using xrdb -q .

**Comment by Fig-R** [comment]
Ok, thanks for your help.
